streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -220,6 +221,50 @@ public void shouldRestoreAll() throws Exception {
             "c");
     }
 
+    @Test
+    public void shouldPutOnlyIfAbsentValue() throws Exception {
+        subject.init(context, subject);
+        final Bytes keyBytes = new Bytes(stringSerializer.serialize(null, "one"));
+        final byte[] valueBytes = stringSerializer.serialize(null, "A");
+        final byte[] valueBytesUpdate = stringSerializer.serialize(null, "B");
+
+        subject.putIfAbsent(keyBytes, valueBytes);
+        subject.putIfAbsent(keyBytes, valueBytesUpdate);
+
+        final String retrievedValue = stringDeserializer.deserialize(null, subject.get(keyBytes));
+        assertTrue(retrievedValue.equals("A"));
+    }
+
+    @Test
+    public void shouldRetrieveFirstValue() throws Exception {
```

Comment:
Hmm.. `first` and `last` of RocksDBStore are not used anywhere, I think it is a leftover of the code clean up introduced in KAFKA-4499 (cc @RichardYuSTUG ). Maybe we can just remove these two functions from RocksDBStore.

Reply:
ack